great_expectations/data_context/data_context.py

```diff
@@ -670,6 +672,78 @@ def update_return_obj(self, data_asset, return_obj):
         else:
             return return_obj
 
+    def profile_datasource(self, datasource_name, profiler_name="PseudoPandasProfiling", max_data_assets=10):
+        # logger.info("Profiling %s with %s" % (datasource_name, profiler_name))
```

Comment:
Simply uncommenting this line works when I run the CLI -- what issue are you having?

Reply:
err... I need to look. It might have been an issue running the CLI through the tests...?

Reply:
Confirmed this works. Also fixed the ability to use different profilers.